dyn/tm/services/dsf.py

```diff
@@ -4142,7 +4142,7 @@ def nodes(self, nodes):
                     _nodeList.append(node)
         elif isinstance(nodes, dict):
             _nodeList.append(nodes)
-        elif isinstance(nodes, DSFNode) or type(node).__name__ == 'Node':
+        elif isinstance(nodes, DSFNode) or type(nodes).__name__ == 'Node':
```

Comment:
I am trying to get rid of the cyclic dependency where tm.zones imports tm.service.dsf  and tm.service.dsf import tm.zones.Node. This was the most elegant solution I could come up with for identifying if a Node object was passed in without adding lots of breaking changes. isinstance() will not work because we are no longer importing Node.